src/test/java/org/springframework/integration/aws/config/xml/SqsOutboundChannedlAdapterParserTest.java

```diff
@@ -0,0 +1,45 @@
+package org.springframework.integration.aws.config.xml;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.beans.factory.BeanDefinitionStoreException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.integration.aws.outbound.SqsMessageHandler;
+import org.springframework.integration.test.util.TestUtils;
+
+/**
+ * @author Rahul Pilani
+ */
+public class SqsOutboundChannedlAdapterParserTest {
+
+    @Test(expected = BeanDefinitionStoreException.class)
+    public void test_sqs_resource_resolver_defined_with_queue_messaging_template() {
+        ApplicationContext applicationContext = new ClassPathXmlApplicationContext("org/springframework/integration/aws/config/xml/SqsOutboundChannelAdapterParserTests-context-bad.xml");
+        applicationContext.getBean("sqsOutboundChannelAdapter");
```

Comment:
This `getBean()` isn't reachable, because we can't even rise an application context.